src/App.tsx

```diff
@@ -1,11 +1,53 @@
-import React from 'react';
-
+import React, { useEffect } from 'react';
+import {
+  Route, Switch, Redirect,
+} from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { Header } from './components/Header/Header';
+import { PhonesPage } from './pages/PhonesPage';
+import { HomePage } from './pages/HomePage';
+import { TabletsPage } from './pages/TabletsPage';
+import { AccessoriesPage } from './pages/AccessoriesPage';
+import { Footer } from './components/Footer/Footer';
+import { FavoritesPage } from './pages/FavouritesPage';
+import { CardPage } from './pages/CartPage';
 import './App.scss';
+import './helpers/fonts/fonts.scss';
+import { loadData } from './store/index';
+import { ProductPage } from './pages/ProductPage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+const App = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(loadData());
+  }, [dispatch]);
+
+  return (
+    <>
+
+      <Header />
+
+      <main>
```

Comment:
don't leave such code, clean it

Reply:
> ![image](https://user-images.githubusercontent.com/33497431/84869702-0e95e700-b087-11ea-881b-573c465f55f7.png)
> add paddings and cursor pointer to select
> 
> ![image](https://user-images.githubusercontent.com/33497431/84869994-66345280-b087-11ea-8612-36d70393dbac.png)
> 768px -> fix styles, the same with mobile view

we did not do adaptive version yet. Waiting for review of desktop version